server/main.go

```diff
@@ -1,3 +1,4 @@
+// Command server is the Instant Share server. It is responsible for accepting uploads and serving those files.
```

Comment:
Eh, okay. You're more familiar than I am when it comes to Go conventions, it just seemed odd.